react/hooks/mapCustomTypes.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+import EnumTable from '../components/EnumTable'
+import ObjectsTable from '../components/ObjectsTable'
+import ArrayTable from '../components/ArrayTable'
+import { capitalizeFirst } from '../utils/capitalizeFirst'
+
+export const mapCustomTypes = (
+  propsObj: Record<string, ObjSchemaInterface>,
+  messages: Record<string, Record<string, string>>
+) => {
+  const mappedTypes: JSX.Element[] = []
+  const mappedProps: Record<string, ObjSchemaInterface> = propsObj
+  for (const key in mappedProps) {
```

Comment:
Use a `.map` function instead, and this function is a bit larger than it should me. Using map and avoiding `let` will help not hat